src/Components/NavBar/Menus/MobileNavMenu/MobileNavMenu.tsx

```diff
@@ -255,9 +257,20 @@ const AuthenticateLinks: React.FC = () => {
 }
 
 const LoggedInLinks: React.FC = () => {
+  const { user } = useSystemContext()
+  const conversationsEnabled = userHasLabFeature(
+    user,
+    "User Conversations View"
+  )
   return (
     <Box>
       <Separator my={1} color={color("black10")} />
+      {conversationsEnabled && (
+        <MobileLink href="/user/conversations">
+          <EnvelopeIcon fill={"black60"} top={3} mr={10} />
```

Comment:
Hmm, the values here feel weird. `top={3}` should be 30 pixels...

That's fine though, we'll fix that in a separate PR